velox/substrait/VeloxToSubstraitPlan.h

```diff
@@ -36,11 +36,13 @@ class VeloxToSubstraitPlanConvertor {
   /// Convert Velox PlanNode into Substrait Plan.
   /// @param vPlan Velox query plan to convert.
   /// @param arena Arena to use for allocating Substrait plan objects.
+  /// @param prefix Registered function prefix.
```

Comment:
Would you clarify what it is used for / applied to?

Reply:
Added more info for it.